deploy/helm/templates/statefulset.yaml

```diff
@@ -66,6 +66,8 @@ spec:
             memory: {{ $container.resources.requests.memory }}
           limits:
             memory: {{ $container.resources.limits.memory }}
+            cpu: 0.5
```

Comment:
The simplest way is to add the cpu and storage limits in the values.yaml file.
I believe the linter will accept that.
```
    resources:
      requests:
        memory: "128Mi"
        cpu: 0.5
        storage: 2GI
      limits:
        memory: "512Mi"
        cpu: 2
        storage: 4Gi
```


worst case,  if the above doesn't work as expected, you can bury the default values in this STS manifest with something like
`cpu:  {{ $container.resources.limits.cpu }} | default 0.5`

Reply:
done